take-homes/fe-take-home.md

```diff
@@ -0,0 +1,29 @@
+# NY Times News Reader Take Home Test
+
+You will create a news reader application that pulls articles from the New York Times API.
+
+## Requirements:
+
+At a minimum, you must show:
+
+* A list of articles 
+* A "detailed" view for each article (as opposed to summary/list view)
+* Articles in the list must link to the detailed article view you create
+* Some sort of navigation, filter, OR sort
+
+Beyond the base requirements, include any additional elements you'd like, just be mindful of your time. You don't need to go overboard, just consider what might be necessary to achieve MVP so that this news feed is easy-to-use, information is clear, and navigation is painless. Put yourself in a user's shoes and think about what you would need (or want). We aren't looking for UI design, but we would like to see some demonstration of CSS knowledge, and the app should be usable on desktop, tablet, and mobile devices.
+
+## Things we'll be looking for:
+
+* a strong understanding of front-end development fundamentals (primarily JavaScript, but also CSS and HTML)
+* an understanding of React js best practices
+* an understanding of basic usability practices and standards
+* your ability to prioritize for MVP
+* a clear hierarchy of information
+* clean, well thought out code
```

Comment:
Just an idea, similar to what we discussed in our last FE meeting, do some of these "learning goals" feel a bit repetitive?  Maybe we could pick the top 3 here?  Such as:

1.  an understanding of React JS best practices
2. your ability to prioritize for MVP
3. Creating an effective UI and UX?

Reply:
I slimmed these down some. I wonder if having more bullets is more instructive in this case?